pkg/loadbalancer/loadbalancer.go

```diff
@@ -572,22 +570,20 @@ func NewBackend(id BackendID, protocol L4Type, ip net.IP, portNumber uint16) *Ba
 	return &b
 }
 
-// NewBackendWithState creates the Backend struct instance from given params,
-// and sets the restore state for the Backend.
+// NewBackendWithState creates the Backend struct instance from given params.
 func NewBackendWithState(id BackendID, protocol L4Type, ip net.IP, portNumber uint16,
```

Comment:
Why are we not setting the weight field to the default value here - `1`? This is called from the restore path.